Persephone/Components/Browser/Album Browser/AlbumViewItem.swift

```diff
@@ -14,26 +14,28 @@ class AlbumViewItem: NSCollectionViewItem {
 
   override var isSelected: Bool {
     didSet {
-      albumCoverBox.layer?.borderWidth = isSelected ? 5 : 0
+      setAppearance(selected: isSelected)
     }
   }
 
   override func viewDidLoad() {
     super.viewDidLoad()
 
     albumCoverView.wantsLayer = true
-    albumCoverView.layer?.cornerRadius = 3
+    albumCoverView.layer?.backgroundColor = NSColor.black.cgColor
```

Comment:
What do you think about writing an extension on...well, in iOS it's `UIView`...so that you can do something like

`albumCoverView.addBorder(.black)` or `albumCoverView.addBorder(.black, width: 5, radius: 6)`

Reply:
I like this. In macOS it would be `NSView`.

Reply:
Actually, thinking about this a little more I'm only setting the width and radius here, then setting the colour in the `setAppearance` function. I don't need to set the width and radius every time so I don't think would save me all that much.

I'll think on it some more though. I already have an `NSView` extension!